pkgs/by-name/ts/tsukimi/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  fetchurl,
+  rustPlatform,
+  cargo,
+  pkg-config,
+  glib,
+  mpv,
+  ffmpeg,
+  libadwaita,
+  gst_all_1,
+  gtk4,
+  openssl,
+  libepoxy,
+  gsettings-desktop-schemas,
+  wrapGAppsHook4,
+  makeDesktopItem,
+  gettext,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tsukimi";
+  version = "0.12.2";
+
+  src = fetchFromGitHub {
+    owner = "tsukinaha";
+    repo = "tsukimi";
+    rev = "v${version}";
+    hash = "sha256-pJ+SUNGQS/kqBdOg21GgDeZThcjnB0FhgG00qLfqxYA=";
+  };
+
+  icon = fetchurl {
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/tsukimi.png?h=tsukimi-git&id=22721d9f058dafeec7f07907aec68fa232ccd2c7";
```

Comment:
What's the problem with the icon included in the repo (https://github.com/tsukinaha/tsukimi/blob/v0.12.2/resources/ui/icons/tsukimi.png)?